### PR TITLE
Chore: favor undefined to null

### PR DESCRIPTION
**Pull request by niedzielski**
https://basarat.gitbooks.io/typescript/docs/tips/null.html
https://github.com/eslint/eslint/issues/6701

### CODE REVIEW

File: src/ElementUtilities.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
  */
 const isNestedInTable = (el) => {
-  return (findClosest(el, 'table') !== null)
+  return (findClosest(el, 'table'))
```
Comment (niedzielski) [inline src/ElementUtilities.js]: Use `return !!findClosest(el, 'table')` to be explicit

File: src/ElementUtilities.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
```
Comment (niedzielski) [inline src/ElementUtilities.js]: Please also update findClosest and other methods to eliminate parentheses usage.

Comment (niedzielski) [inline src/ElementUtilities.js]: This probably deviates from the original implementation in that it only considers the parent, not the passed in element. Consider replacing with the original and replacing usages with el.parentElement OR renaming to findClosestAncestor().

Comment (niedzielski) [inline src/ElementUtilities.js]: Also, regardless: add a test for the case of whether the element itself can / is expected to be returned when it matches.

Comment (niedzielski) [inline src/ElementUtilities.js]: On second thought, just go with renaming and adding that test and updating the assignment / parentheses usage.

Comment (niedzielski) [inline src/ElementUtilities.js]: Do the same thing for other methods elsewhere including WidenImage.widenAncestors().

Comment (niedzielski) [inline src/ElementUtilities.js]: Please put in separate PR yo.

File: src/ElementUtilities.js (hunk 1 above)
Comment (niedzielski) [inline src/ElementUtilities.js]: Fixed!

File: src/ElementUtilities.js (hunk 2 above)
Comment (niedzielski) [inline src/ElementUtilities.js]: Ok, this is all done but it landed in this PR :]

**Comment by montehurd** [comment]
I'm a bit worried about the readability trade-offs now that I see the diff... the for loops and `!!` are less clear at first glance, but I'm not sure if that's just because I'm used to the old way... I'll take another peek tomorrow when I'm not sleepy :)
